.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,46 @@
+# Proposed change
```

Comment:
**For your consideration; totally optional**

You may want to consider adopting a more formal commit taxonomy like Conventional Commits: https://www.conventionalcommits.org/en/v1.0.0/#summary

If you get this right, you can progress to Semantic Releases: https://semantic-release.gitbook.io/semantic-release/

Reply:
**For all the rest of my comments**, feel free to replace "this contribution" with "my changes" or "these changes" or "this submission" or "this change".

Reply:
That would be great. I have seen a lot of initiatives in this space, but Angular Commit Message Conventions (on which semantic-release is based) stands out by being clean and straight forward 👍